web/js/index.js

```diff
@@ -22,7 +22,26 @@ const loadTexture = (path, loader, onProgress) => {
 }
 
 // Create scene and camera
-let scene = new THREE.Scene();
+var scene;
```

Comment:
Instead of using _var_ here, we should use _let_. The only issue with var is that it can be used outside of its own scope. Since this is global we shouldn't have problems, but it's a general guideline to use _let_ and _const_ instead of var.

Reply:
Updated file- changed var to let.